iib/workers/tasks/build.py

```diff
@@ -117,6 +118,16 @@ def _build_image(dockerfile_dir: str, dockerfile_name: str, request_id: int, arc
         {'cwd': dockerfile_dir},
         exc_msg=f'Failed to build the container image on the arch {arch}',
     )
+    log.debug('Verifying that %s was built with expected arch %s', destination, arch)
+    buildah_inspect = json.loads(run_cmd(['buildah', 'inspect', destination]))
+    archmap = {"amd64": "x86_64", "arm64": "aarch64", "s390x": "s390x", "ppc64le": "ppc64le"}
```

Comment:
I wonder if it makes sense to move this to worker config instead. That way, it's easier to maintain and we don't forget about it if an arch specific change is made in the future.

Reply:
sounds fair